README.md

```diff
@@ -1 +1,22 @@
-# beyond08-group1
\ No newline at end of file
+# Tasties
+![WhatsApp Image 2022-11-16 at 5 32 47 PM](https://user-images.githubusercontent.com/81830917/202234557-2cdf76cb-fba4-48c9-8e96-b1e094b685cb.jpeg)
```

Comment:
I would upload the image to the repo. You can take a look [here](https://github.com/redhat-beyond/ResuMe/blob/main/README.md) for example

Reply:
@WaseemAbuSobe

Reply:
@WaseemAbuSobe, Please upload the image I sent in the Slack channel.